### ISSUE TITLE
No Folder with German Umlaute ä,ö,ü

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#51]
```
Folders with German Umlaute ä,ö,ü do not work properly. After reload the
notebook not all Folder will be recovered.

```

Original issue reported on code.google.com by `jspille1...@gmail.com` on 19 May 2009 at 8:00

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in KeepNote 0.5.3.  Thanks for your report.
```

Original comment by `matt.rasmus@gmail.com` on 27 May 2009 at 1:20
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 8 Jul 2009 at 12:42
- Changed state: **Done**
